### PR TITLE
Fix grammar and typos

### PR DESCRIPTION
**Pull request by l0l869**
I didn't want to change too much, so I only did some minor fix-ups. However, I do believe some parts could benefit with a rewrite to enhance clarity.

### CODE REVIEW

File: fullgame/hundo.md [hunk 1]
```diff
@@ -8,25 +8,33 @@
...
- * You are allowed to segment your run as long as you do not go in the world off recording/stream and you do not gain any achievements anywhere other than in the world on recording/stream. You are required to show the achievements at the start and at the end of every segment.
- * You must play on the version that the version splits are named after, for example for a 1.16 100% you can play on 1.16.0, 1.16.1, 1.16.100 etc (You cannot complete a 1.16 100% run on 1.19 by only completing the necessary achievements for 1.16).
-* If one of those 4 achievements don't pop up, even though you did them,
-  you can put a time stamp of when you did it instead, you still have to do them
+* You are allowed to segment your run as long as you do not go in the world off
+recording/stream and you do not gain any achievements anywhere other than in
+the world on recording/stream. You are required to show the achievements at the
+start and at the end of every segment.
+* You must play on the version that the version splits are named after. For
+example, for a 1.16 100%, you can play on 1.16.0, 1.16.1, 1.16.100, etc.
```
Comment (AnInternetTroll) [inline fullgame/hundo.md]: should be 1.16 100% run 
(both us and lol missed this)
```suggestion
example, for a 1.16 100% run, you can play on 1.16.0, 1.16.1, 1.16.100, etc.
```

File: global/README.md [hunk 2]
```diff
@@ -8,61 +8,66 @@
 
 ## General Rules
 
-* The use of external tools that modify the game in any way, shape, or form are
+* The use of external tools that modify the game in any way, shape, or form is
 forbidden.
-* The as of now Android-exclusive 1.2.6.* duping glitch is banned from all
-categories unless explicitely stated otherwise.
+* The, as of now, Android-exclusive 1.2.6.* duping glitch is banned from all
```
Comment (AnInternetTroll) [inline global/README.md]: unrelated to the changes but this should be Mobile-exclusive (iOS can change versions, and has 1.2.6 available (https://70.glitch.me/)
```suggestion
* The, as of now, Mobile-exclusive 1.2.6.* duping glitch is banned from all
```

File: global/README.md [hunk 3]
```diff
@@ -8,61 +8,66 @@
...
-	- This includes changing gamemodes.
-* Reloading the world (quitting and loading the same world again) always puts the run into the glitched category.
+	- This includes changing game modes.
+* Reloading the world (quitting and loading the same world again) always puts
+the run into the glitched category.
 * While we do promote ingenuity and creativity from runners, do not attempt to
 be cheeky in regards to the rules. Exploiting loopholes in the rules that were
 clearly not intended to be exploited may result in your run being rejected.
-	- If you do find a loophole, we ask you point it out to a moderator instead.
+	- If you do find a loophole, we ask taht you point it out to a moderator
```
Comment (AnInternetTroll) [inline global/README.md]:
```suggestion
	- If you do find a loophole, we ask that you point it out to a moderator
```

File: global/README.md [hunk 4]
```diff
@@ -8,61 +8,66 @@
...
+	- Full game runs performed on beta versions of the game must be submitted
 	to the beta category on our category extensions.
-	- Beta versions of the game are not permitted for individual level runs.
+	- Beta versions of the game are not permitted for Individual Level runs.
 * If you submit a video significantly longer than the duration of the run,
-specify timestamps of the runs beginning and end in the submission description.
-	- This is not a required rule, however it is strongly recommend as it
+specify timestamps of the run's beginning and end in the submission
+description.
+	- This is not a required rule; however, it is strongly recommend as it
```
Comment (AnInternetTroll) [inline global/README.md]:
```suggestion
	- This is not a required rule; however, it is strongly recommended as it
```

**Review by AnInternetTroll — CHANGES_REQUESTED** [review]
I've left Khalooody's review here as he is not too familiar with GitHub's interface.

File: catext/any-peaceful-ic.md [hunk 5]
```diff
@@ -9,8 +9,8 @@ Any% Peaceful is exclusive (as of now) to Android due to the glitch needed to
 run the category. The current duping method is done on Android version 1.2.6.1
 (1.2.6.60) or 1.2.6.55.
 
-* The glitch used is allowed on Any% Peaceful (Item Creation) ONLY. Not on any
-other leaderboards unless explicitely stated otherwise.
-* Timing ends the frame the screen begins to fade after exiting the end.
+* The glitch used is ONLY allowed on Any% Peaceful (Item Creation). It is not
+permitted on any other leaderboards unless explicitly stated otherwise.
+* Timing ends (on the frame) when the End Poem fades in upon exiting the End.
```
Comment (AnInternetTroll) [inline catext/any-peaceful-ic.md:14]: Capitalizing "End" here is kinda weird. I'm not sure if "the end" or "The End" is correct here, but I think "the End" is certainly odd

For reference the minecraft blog spells it as "The End"
https://www.minecraft.net/en-us/article/around-block--end-highlands
```suggestion
* Timing ends (on the frame) when the End Poem fades in upon exiting The End.
```

File: catext/nether.md [hunk 6]
```diff
@@ -6,6 +6,7 @@
 [Back](../README.md)
 
 * Must enter the nether.
-* Timing ends the frame the loading screen appears after entering the nether.
+* Timing ends (on the frame) when the loading screen appears upon entering the
+nether.
```
Comment (AnInternetTroll) [inline catext/nether.md]: This isn't even related to your change, but while at it this should also be capitalized
https://www.minecraft.net/en-us/updates/nether
```suggestion
* Timing ends (on the frame) when the loading screen appears upon entering The
Nether.
```

File: catext/ssp-coop.md [hunk 7]
```diff
@@ -5,5 +5,5 @@
 
 [Back](../README.md)
 
-* All [SPP Solo rules](ssp.md) apply to this category.
+* All [SSP Solo rules](ssp.md) apply to this category.
```
Comment (AnInternetTroll) [inline catext/ssp-coop.md:8]: It's pretty odd that we don't seem to define "SSP" or what it is (Set Seed Peaceful) here. I'm not sure if this should be addressed in this change or not, but in any case we should follow it up, as I imagine the heavy use of abbreviations can be confusing

File: cheaters/README.md [hunk 8]
```diff
@@ -12,34 +12,33 @@
...
 player has cheated in a game outside of Bedrock Edition, action will be taken.
 	- Players caught cheating in a game outside of the Minecraft series are
-	generally placed on the suspicious players list, but allowed to submit
+	generally placed on the suspicious players list but allowed to submit
 	runs.
-	- Players caught cheating in other games within the Minecraft series
-	such as Java Edition and Legacy Edition will generally be banned from
-	our leaderboards.
+	- Players caught cheating in other games within the Minecraft series,
+	such as Java Edition and Legacy Edition, will generally be banned from our
```
Comment (AnInternetTroll) [inline cheaters/README.md]: Nitpick, but it sounds bad to use "our" here. The leaderboards are "owned" by the community, not by whoever writes the rules
```suggestion
	such as Java Edition and Legacy Edition, will generally be banned from the
```

File: fullgame/any-glitchless.md [hunk 9]
```diff
@@ -7,6 +7,6 @@
 
 * Beat the game as fast as possible without the use of glitches.
 
-* Timing ends the frame the screen begins to fade after exiting the end.
+* Timing ends (on the frame) when the End Poem fades in upon exiting the End.
```
Comment (AnInternetTroll) [inline fullgame/any-glitchless.md:10]:
```suggestion
* Timing ends (on the frame) when the End Poem fades in upon exiting The End.
```

File: fullgame/any.md [hunk 10]
```diff
@@ -9,9 +9,9 @@
 * The use of glitches is required.
 	- If you did not use glitches, submit to
 	[Any% Glitchless](any-glitchless.md).
-	- Any glitches used must provide some sort of advantage, performing a
+	- Any glitches used must provide some sort of advantage; performing a
 	visual bug that does not affect gameplay to submit your glitchless run
 	to glitched is forbidden.
-* Timing ends the frame the screen begins to fade after exiting the end.
+* Timing ends (on the frame) when the End Poem fades in upon exiting the End.
```
Comment (AnInternetTroll) [inline fullgame/any.md:15]:
```suggestion
* Timing ends (on the frame) when the End Poem fades in upon exiting The End.
```

File: fullgame/bosses-glitchless.md [hunk 11]
```diff
@@ -17,24 +17,26 @@
...
 
-* The same rules apply as All Bosses, however you are only required to kill
+* The same rules apply as for All Bosses, however you are only required to kill
 the Elder Guardian.
 
 ## Warden
 
-* The same rules apply as All Bosses, however you are only required to kill the Warden
+* The same rules apply as for All Bosses, however you are only required to kill
+the Warden.
```
Comment (AnInternetTroll) [inline fullgame/bosses-glitchless.md]: I am not sure how to feel about this, but "the warden" is lowercased on minecraft.net (as opposed to "The End" or "the Wither")

https://www.minecraft.net/en-us/article/meet-warden
```suggestion
the warden.
```

**Review by AnInternetTroll — CHANGES_REQUESTED** [review]
Here is my own review.

Overall these changes are certainly a welcome improvement. Thanks for putting in the time to quality check the rules!

File: global/README.md (hunk 3 above)
Comment (l0l869) [reply global/README.md]: I'm not sure how I managed to do taht.

File: catext/any-peaceful-ic.md (hunk 5 above)
Comment (l0l869) [reply catext/any-peaceful-ic.md:14]: "The" is only capitalised when it's at the start of a sentence or is part of a title.

In the blog you referrenced, there are instances of "the End" as opposed to "The End". The only time "The End" appears is at the start of a sentence or as a title. In the context: "The first, confusingly, is called **The End** – that’s the bit where you fight the Ender Dragon." it is referred to as a title.

You can see it with "The Nether" here:
https://www.minecraft.net/en-us/article/the-bedrock-nether-update-beta-here-

File: fullgame/bosses-glitchless.md (hunk 11 above)
Comment (l0l869) [reply fullgame/bosses-glitchless.md]: I agree with this.

**Comment by AnInternetTroll** [comment]
Oh my you're right. I misread that. Ignore this comment then!

**Review by AnInternetTroll — APPROVED** [review]
LGTM. I'll wait to see if anyone else has anything left to say, but otherwise I think we can merge this just fine

**Pull request merged**